src/engine/Server.cpp

```diff
@@ -492,6 +504,43 @@ Awaitable<void> Server::process(
       parsedHttpRequest.operation_);
 }
 
+// ____________________________________________________________________________
+bool containsParam(const auto& params, const std::string& param,
+                   const std::string& expected) {
+  auto parameterValue =
+      ad_utility::url_parser::getParameterCheckAtMostOnce(params, param);
+  return parameterValue.has_value() && parameterValue.value() == expected;
```

Comment:
```suggestion
  return parameterValue == expected;
```
It does the right thing and it is more concise.